### PR TITLE
[frontend] Update selectors

### PR DESCRIPTION
**Pull request by matiasmoreno876**
|Related issue|
|---|
| [1412](https://github.com/wazuh/wazuh-qa/issues/1412) |

## Description
Hi team, in this PR I updated the selectors for Wazuh Menu.

### CODE REVIEW

**Review by MauGaP — APPROVED** [review]

File: tests/frontend/test_kibana_app/cypress/integration/step-definitions/management/rules/navigate-to-rules.when.js [hunk 1]
```diff
@@ -3,12 +3,12 @@ import { clickElement, getObject } from '../../../utils/driver';
...
-  rulesButton,
+  rulesLink,
 } from '../../../pageobjects/wazuh-menu/wazuh-menu.page';
 
 When('The user navigates to rules', () => {
   clickElement(wazuhMenuButton);
   clickElement(managementButton);
-  clickElement(rulesButton);
+  clickElement(rulesLink);
   cy.wait(3000);
```
Comment (gabiwassan) [inline tests/frontend/test_kibana_app/cypress/integration/step-definitions/management/rules/navigate-to-rules.when.js:13]: This `cy.wait()` its neccessary?

**Review by gabiwassan — APPROVED** [review]
LGTM :)

File: tests/frontend/test_kibana_app/cypress/integration/step-definitions/management/rules/navigate-to-rules.when.js (hunk 1 above)
Comment (MauGaP) [reply tests/frontend/test_kibana_app/cypress/integration/step-definitions/management/rules/navigate-to-rules.when.js:13]: it is not, but I already deleted all of them, and merged to `feature/frontend`

Maybe, he needs to pull `feature/frontend` into his branch

Comment (matiasmoreno876) [reply tests/frontend/test_kibana_app/cypress/integration/step-definitions/management/rules/navigate-to-rules.when.js]: Yes, as @MauGaP mentioned, I needed to update my branch.

**Review by frankeros — APPROVED** [review]
LGMT!

**Pull request merged**